app/pathogen/arbovirus/analyze/columns.tsx

```diff
@@ -238,43 +238,43 @@ export const columns: ColumnDef<Estimate>[] = [
       );
     },
     cell: ({ row }) => {
-      const end_date = row.original.sample_end_date;
+      const end_date = row.original.sampleEndDate;
       if(end_date) {
         return TranslateDate(end_date)
       }
       return 'N/A';
     }
   },
   {
-    accessorKey: "who_region",
+    accessorKey: "whoRegion",
```

Comment:
Alright so there's actually one tiny functionality difference here. The names of the columns in the dropdown and the csv downloaded are in camel case now instead of snake case. These names should really be "Sample Size" instead of "sample_size" or "sampleSize". I didn't make the change to keep this PR small so if everyone else is on board to hit this later, let's hit it later. Otherwise, just ask and I'll open a different PR that we can merge before this one that changes the names of the columns to be more friendly. I'm pretty flexible hahaha

![image](https://github.com/PathoTracker/Pathotracker/assets/86806388/c33f4c8c-e750-46bd-abe9-d06e1ca39759)

Reply:
Will do! :sparkles: